core/src/main/kotlin/com/alecstrong/sql/psi/core/sql.bnf

```diff
@@ -234,7 +234,7 @@ literal_value ::= ( numeric_literal
                   | NULL
                   | CURRENT_TIME
                   | CURRENT_DATE
-                  | CURRENT_TIMESTAMP ) {
+                  | CURRENT_TIMESTAMP [ '(' signed_number ')' ] [ ON UPDATE CURRENT_TIMESTAMP [ '(' signed_number ')' ] ] ) {
```

Comment:
This update should be in MySql.bnf, but I had some trouble getting it to work, possibly cause of the mixin?

Reply:
overriding `literal_value` didn't work?

Does it give an exception?

Reply:
Let me update locally and push so you can see